tests/resources/example_fetch_instrument_response.json_entity.json

```diff
@@ -0,0 +1,294 @@
+[
+  {
+    "pk": 4,
+    "tableName": "Instrument",
+    "columns": [
+      {
+        "datatype": "STRING",
+        "name": "nstr_name",
+        "title": "Name",
+        "position": 0,
+        "value": "323_EPHYS1_OPTO",
+        "hidden": false,
+        "editable": true
+      },
+      {
+        "datatype": "ENUM",
+        "name": "icon",
+        "title": "Icon",
+        "position": 1,
+        "value": "data_icons/rat.png",
+        "hidden": false,
+        "editable": true,
+        "displayValue": "data_icons/rat.png"
+      },
+      {
+        "datatype": "STRING",
+        "name": "nstr_uniqueIdentifier",
+        "title": "Unique identifier",
+        "position": 2,
+        "value": "nstr_323_dot_rig1",
+        "hidden": false,
+        "editable": true
+      },
+      {
+        "datatype": "STRING",
+        "name": "nstr_description",
+        "title": "Description",
+        "position": 3,
+        "value": "<p>EPHYS1 Rig configuration for opto experiments</p>\n\n<p>Main User: Anna Lakunina</p>\n",
```

Comment:
I try to scrub out real names in test data sets

Reply:
Resolved in: [b9a30bb](https://github.com/AllenNeuralDynamics/aind-slims-api/pull/16/commits/b9a30bb39637bb3441e5c9ccf59afdd3d1d15e8e)